namespaces/live.cloud-platform.service.justice.gov.uk/care-arrangement-plan-prototype/03-resourcequota.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: namespace-quota
+  namespace: care-arrangement-plan-prototype
+spec:
+  hard:
+    pods: "10"
```

Comment:
@josephsilcock I noticed this was a bit high and notched it down. Otherwise its a direct copy/past from your PR 🚀

Reply:
Wonderful, thanks so much for your help!

Reply:
No probs, shortly there will be an automated reply in this issue.

I doubt you'll have permissions to see the result, but I'll keep an eye out and share the outcome.